cmd/rhelemeter-server/README.md

```diff
@@ -0,0 +1,75 @@
+# Rhelemeter
+
+Rhelemeter is a Prometheus remote write forward proxy. It allows RHEL hosts or an intermediate proxy to push metrics to
+a upstream Prometheus (or Prometheus remote write compatible) endpoint.
+
+[rhelemeter-server](https://github.com/openshift/telemeter/tree/master/cmd/rhelemeter-server) needs to receive
+and send metrics across security boundaries, and thus performs additional authorization and data integrity checks as required.
+
+Architecture
+```mermaid
+C4Deployment
+    title rhelemeter-server
+
+    Deployment_Node(rhel-os, "RHEL OS VM", ""){
+        Container(rhel, "Prometheus Remote Write Client", "", "Writes host OS based metrics.")
+    }
+
+    Deployment_Node(proxy, "Intermediate Proxy", ""){
```

Comment:
I would name the proxy, would be easier to understand. `Intermediate Proxy (Akamai)`

Reply:
I purposely left this lacking lots of detail. The intention was more to describe config and traffic flow. We have a requirement to created a detailed internal architecture document for this and other components so I would prefer to push it to there. Does that make sense?

Reply:
Yes makes sense 👍